### DISCUSSION TITLE
www.wlcb-db.com/login  web site not working error code 500

### DISCUSSION THREAD

**Discussion by ghost** [AzuraCast/AzuraCast#7250] [General]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### What happened?

cannot sign in to my website for broadcasting https://www.wlcp-db.com/login  With this URL I get error code 500.

**Comment by ghost** [comment]
All 12 stations are not working.  Cannot sign in to determine issue.

**Comment by renatopinto89** [comment]
Have you tried the good old reliable "turn it off and back on" method?
Maybe a full server reboot can fix that

Or `docker compose down` and a `docker compose up -d`

**Comment by ghost** [comment]
I can't sign in to do the reboot.  Is there a way to do the reboot without signing in?

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by BornKillaz** [comment]
You can restart AzuraCast with API, but you need an API key fot that. 

As already suggested, login as root and do a server reboot, or restart the container, or do a  backup and reinstall AzuraCast.

Sometimes this type of issues happens when you install a Rolling Release version a then go back to the Sable version.

**Comment by Vaalyn** [comment]
I moved this thread over to discussions as this is not an issue with AzuraCast itself

**Reply by ghost** [reply]
- [ ] I did a reboot through linode.   Still not working.  I just did a full backup before having this problem.  Would I run an install of Azuracast and then restore from backup?

**Reply by ghost** [reply]
What is a restart container and how is it performed?

**Comment by ghost** [comment]
I need help.  I have 12 stations and all are down.  I have done a reboot through linoleum,  still not working

**Reply by ghost** [reply]
Linode

**Reply by BusterNeece** [reply]
The community will do its best to help as they're available, but keep in mind that this is free and open-source software and we are extremely overloaded with bug reports, feature requests and other requests for help. Your patience is appreciated.

**Reply by ghost** [reply]
I understand.  I become a monthly supporter today to help financially
